### PR TITLE
SA-2095: Do not display report builder empty state for subaccount reporting users

### PR DESCRIPTION
**Pull request by bkemper**
### What Changed

- Removed Report Builder empty state, for now, see Slack conversation in uxfe channel

### How To Test

- See steps to reproduce in bug ticket

### CODE REVIEW

File: src/pages/reportBuilder/ReportBuilder.js [hunk 1]
```diff
@@ -466,14 +474,16 @@ export function ReportBuilder({
 
 // Redux
 const mapStateToProps = state => ({
+  canManageDomains: hasGrants('sending_domains/manage'),
```
Comment (kchu1337) [inline src/pages/reportBuilder/ReportBuilder.js:477]:
```suggestion
  canManageDomains: hasGrants('sending_domains/manage')(state),
```

File: src/pages/reportBuilder/ReportBuilder.js [hunk 2]
```diff
@@ -466,14 +474,16 @@ export function ReportBuilder({
...
   chart: state.summaryChart,
+  isOnPremise: hasAccountOptionEnabled('allow_events_ingest')(state),
+  isSubaccoutReporting: hasRole(ROLES.SUBACCOUNT_REPORTING),
```
Comment (kchu1337) [inline src/pages/reportBuilder/ReportBuilder.js:480]:
```suggestion
  isSubaccoutReporting: hasRole(ROLES.SUBACCOUNT_REPORTING)(state),

```

**Review by kchu1337 — CHANGES_REQUESTED** [review]

**Review by kchu1337 — APPROVED** [review]

**Comment by nicklemmon** [comment]
I hate to be that guy, but should have some tests to back up this change?

**Comment by bkemper** [comment]
> I hate to be that guy, but should have some tests to back up this change?

👏 👏 👏 Thank you for holding us accountable!  It is our responsibility to put some test coverage around this fix.  Kevin and I discussed it last night and I forgot to add a todo item to the description.

**Review by kchu1337 — APPROVED** [review]

**Pull request merged**